pom.xml

```diff
@@ -78,15 +78,15 @@
     <!-- matching armeria/grpc/zipkin -->
     <zipkin.groupId>io.zipkin.zipkin2</zipkin.groupId>
     <!-- when updating, update docker/Dockerfile and storage/src/test/java/zipkin2/storage/kafka/IT* -->
-    <zipkin.version>3.0.5</zipkin.version>
+    <zipkin.version>3.0.6</zipkin.version>
     <zipkin-reporter.version>3.2.1</zipkin-reporter.version>
     <spring-boot.version>3.2.2</spring-boot.version>
     <jackson.version>2.16.1</jackson.version>
-    <!-- armeria.groupId allows you to test feature branches with jitpack -->
+    <!-- This allows you to test feature branches with jitpack -->
```

Comment:
at some point I will remove all this stuff as I think jitpack is not really healthy (at least last time I tried it)